conventions.md

```diff
@@ -1,3 +1,58 @@
 # Cryptic Conventions
-You can find the conventions in our official Wiki:
-- [Konventionen (Deutsch)](https://wiki.cryptic-game.net/books/einf%C3%BChrungen/page/konventionen)
+
+ Please use English for all Cryptic-related GitHub posts (in issues, pull requests, commits, etc) and follow the conventions below to ensure a clear and uniform style.
+
+## Branch Naming
+
+The master (or main) branch contains the actual version. New commits to are only added via pull requests which requires two approvals. Learn more about [GitHub branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches).
+
+**Feature branches** contain one or more commits to address one related feature or issue. Prefix your feature branch with `issue/issue-id` before you add the actual branch name. Example: `issue/\<id\>-\<add-my-feature\>`. To update any project dependencies, name your branch like `dependencies/\<dependencies-name\>`.
+
+## Commit Messages
+
+Write short (< 50 characters) **commit messages** in English and use the scheme `<message> (#<issue-id>)`. Do not use just `<em>bug fix</em>` but useful text.
```

Comment:
Does <message> (#<issue-id>) make sense? The final merge is related to the PR which is related to the issue anyway.